source/dlangbot/github.d

```diff
@@ -51,6 +51,11 @@ string markdownEscape(string desc)
     return app.data;
 }
 
+bool usesGitHubIssues(in ref PullRequest pr)
+{
+    return pr.repoSlug.among("dlang/phobos");
```

Comment:
It looks like this needs to be compared against 0, since `among` returns 0 if it's not found and a one-based index of which one was found if one is found.